vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -9395,7 +9450,8 @@ unit:KiloBYTE-PER-SEC
   a qudt:DerivedUnit ;
   a qudt:Unit ;
   dcterms:description "A kilobyte per second (kByte/s) is a unit of data transfer rate equal to 8,000 bits per second, or 1,000 bytes per second, or 8 kilobits per second."^^rdf:HTML ;
```

Comment:
Need to change to say 1024 bytes per second or 8096 bits per second.

Reply:
Changed per this review